### ISSUE TITLE
about training dataset format

### ISSUE THREAD

**Issue by xsy123xsy** [zsyOAOA/DANet#4]
I want to train the model on the cloud. Considering the storage space, can I train the model directly with the picture in png/jpg format instead of hdf5 format.

**Comment by zsyOAOA** [comment]
Of course, you can. However, you have to rewrite the code of loading data.
